test/services/audit/AuditServiceSpec.scala

```diff
@@ -116,21 +142,24 @@ trait AuditTestSupport extends CustomExportsBaseSpec {
 
   val auditFailure = Failure("Event sending failed")
 
-  val auditService = new AuditService(mockAuditConnector, appConfig)
+  val auditService = new AuditService(mockAuditConnector, appConfig)(global)
 
-  def mockSendEvent(evenToAudit: DataEvent = event, result: AuditResult = Success) =
+  def mockSendEvent(
+    evenToAudit: DataEvent = event,
```

Comment:
There is a typo. I think you meant `eventToAudit`

Reply:
not mine, fixed